src/instructlab/rag/convert.py

```diff
@@ -175,7 +176,13 @@ def _initialize_docling():
         artifacts_path=docling_model_path,
         do_ocr=False,
     )
-    ocr_options = _resolve_ocr_options()
+
+    # deactivate MPS acceleration on Github CI for MacOS
```

Comment:
This comment probably belongs, if at all, to github workflow file

Reply:
Thanks Ihar for finding the reference to https://github.com/actions/runner-images/issues/9918

Jaideep would you mind adding a comment with that URL as well, so that we have information a year from now, so we can determine if/when we can drop this workaround.

Reply:
added the URL and moved the comment 👍